content/docs/2.5/scalers/postgresql.md

```diff
@@ -75,8 +75,12 @@ You can authenticate by using a password or store the password within the connec
 
 **Password Authentication:**
 
-- `password` - Password for configured user to login to postgreSQL database
-variables.
+- `host:` Service URL to postgresql. Note that you should use a fully qualified URL -- including the namespace -- as KEDA will need to contact postgresql from a different namespace.
+- `userName:` Username for postgresql user
+- `password` Password for configured user to login to postgreSQL database variables.
+- `port:` Postgresql port
+- `dbName:` Postgresql Database name
+- `sslmode:` SSL policy for communicating with database
```

Comment:
```suggestion
- `sslmode`: SSL policy for communicating with database
```

Is the field `sslmode` or `sslMode`?

Reply:
In the connection string is `sslMode` but our trigger variable is [`sslmode`](https://github.com/kedacore/keda/blob/main/pkg/scalers/postgresql_scaler.go#L105)

Reply:
Can you elaborate on that please? You mean trigger metadata uses `sslMode` but trigger authentication uses `sslmode`?

Reply:
No, the other way around probably?

Reply:
The required key in the connection string is `sslMode`, but inside the scaler we are looking for `sslmode` when we are parsing the metadata. Our key is `sslmode`, we take the value from the metadata key `sslmode` and place in the connection string key `sslMode`

Reply:
Maybe we can look for both when we are parsing the scaler metadata and deprecate `sslmode` in favor of `sslMode` (the last one is how we have to place in connection string)

Reply:
If `sslMode` is only in the connection string then we can just keep on using `sslmode` as we do today, no need to check for `sslMode` field then as well

Reply:
okey, so basically we can let it as it is :)